builder-config.yaml

```
@@ -2,25 +2,25 @@ dist:
   name: castai-collector
   description: CAST AI OTel Collector that outputs Audit Logs to the console
   output_path: ./castai-collector
-  otelcol_version: 0.91.0
+  otelcol_version: 0.103.0
```

Why not 0.104.0 as well?
```suggestion
  otelcol_version: 0.104.0
```

---

missed this one, Thanks!